app/components/TransactionModal.tsx

```diff
@@ -71,19 +85,26 @@ export const TransactionModal = ({
           </div>
         </DialogBody>
         <DialogFooter>
-          <Button
-            width="full"
-            colorScheme={isLoading ? "primary" : "success"}
-            variant="pastel"
-            onClick={() =>
-              window.open(`https://gnosisscan.io/tx/${txHash}`, "_blank")
-            }
-            size="lg"
-          >
-            <>
-              View in explorer <Icon name="arrow-top-right" />
-            </>
-          </Button>
+          {txHash && (
+            <a
+              href={`https://gnosisscan.io/tx/${txHash}`}
+              target="_blank"
```

Comment:
Please prefer using `noopener` and `noreferrer` as extra security layers when using `target="_blank"`

https://medium.com/@halilatilla/target-blank-rel-noopener-noreferrer-b43e9ec05ade

Reply:
Yes, we should add the rel. Thanks